flake.nix

```diff
@@ -0,0 +1,97 @@
+{
+  description =
+    "eclair-lang: An experimental and minimal Datalog that compiles to LLVM";
+  inputs = {
+    np.url = "github:nixos/nixpkgs?ref=haskell-updates";
+    fu.url = "github:numtide/flake-utils?ref=master";
+    hls.url = "github:haskell/haskell-language-server?ref=master";
+    shs.url = "github:luc-tielen/souffle-haskell?ref=master";
+  };
+  outputs = { self, np, fu, hls, shs }:
+    with fu.lib;
+    eachSystem [ "x86_64-linux" ] (system:
+      let
+        config = { };
+        overlay = final: _:
+          with final;
+          with final.haskell.lib;
+          with final.haskellPackages.extend (final: super:
+            let
+              lhsGit = fetchFromGitHub {
```

Comment:
`lhs`? Means llvm-hs? I would fully write it out, because `lhs` is also sometimes used to mean "left-hand side".